docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md

```diff
@@ -0,0 +1,313 @@
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new UnzerNotificationResource(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Create a cart by sending the `POST https://glue.mysprykershop.com/carts`.
+2. Add at least one item to the cart by sending `POST https://glue.mysprykershop.com/{{cart_uuid}}/items`.
+3. Check result by sending the `POST https://glue.mysprykershop.com/checkout-data?include=payment-methods` request.
+
+**Request Example:**
+```json
+{
+  "data":
+  {
+    "type": "checkout-data",
+    "attributes":
+    {
+      "idCart": "{{cart_uuid}}",
+      "payments": [
+        {
+          "paymentMethodName": "Unzer Sofort",
+          "paymentProviderName": "Unzer"
+        }
+      ]
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
+
+Ensure that the request body differs for each Unzer payment method:
+- Property `paymentMethodName` of `payments` must be replaced by the used method—for example, `Unzer Sofort` or `Unzer Credit Card`. 
+- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, and `Unzer Bank Transfer`.
+
+{% endinfo_block %}
+
+<details><summary markdown='span'>Response example:</summary>
+```json
+{
+  "data": {
+    "type": "checkout-data",
+    "id": null,
+    "attributes": {
+      "unzerPublicKey": "string",
+      ...
+    },
+    "links": {
+      "self": "https://glue.mysprykershop.com/checkout-data?include=payment-methods"
+    },
+    "relationships": {
+      "payment-methods": {
+        "data": [
+          {
+            "type": "payment-methods",
+            "id": "1"
+          }
+        ]
+      }
+    },
+    ...
+  },
+  "included": [
+    {
+      "type": "payment-methods",
+      "id": "1",
+      "attributes": {
+        "paymentMethodName": "Unzer Sofort",
+        "paymentProviderName": "Unzer",
+        "priority": 1,
+        "requiredRequestData": [
+          "paymentMethod",
+          "paymentProvider",
+          "unzerPayment.paymentResource.id"
+        ]
+      },
+      "links": {
+        "self": "https://glue.mysprykershop.com/payment-methods/1"
+      }
+    },
+    ...
+  ]
+}
+```
+
+4. Check result by sending the `POST https://glue.mysprykershop.com/checkout` request.
+
+**Request Example:**
+```json
+{
+  "data": {
+    "type": "checkout",
+    "attributes": {
+      "idCart": "{{cart_uuid}}",
+      "payments": [
+        {
+          "paymentMethodName": "Unzer Credit Card",
+          "paymentProviderName": "Unzer",
+          "paymentSelection": "unzerCreditCard",
+          "unzerCreditCard": {
+            "paymentResource": {
+              "id": "{{payment_resource}}"
+            }
+          },
+          "amount": "{{amount}}"
+        }
+      ],
+      ...
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
```

Comment:
If the steps listed in this infoBox are mandatory, then replace `{% info_block infoBox %}` with  `{% info_block warningBox %}`